tests/components/tuya/test_config_flow.py

```diff
@@ -120,14 +120,14 @@ async def test_abort_on_invalid_credentials(hass, tuya):
     )
 
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
-    assert result["errors"] == {"base": "auth_failed"}
+    assert result["errors"] == {"base": "invalid_auth"}
```

Comment:
In my commit I replaced hardcoded strings with constants from config_flow.py.

Reply:
I don't understand the comment. What is the commit and what's the question?

Reply:
I did same work but a bit later. So I closed my PR. But I replaced the strings with constants from config_flow.py
https://github.com/home-assistant/core/pull/41262/commits/aa414d5cd71ee5208b817f590e7a41b779284cf6

Reply:
May be I'm wrong but my opinion is that using "reference string" override the requirement to do this.